test/parallel/test-stream-transform-split-highwatermark.js

```diff
@@ -2,9 +2,9 @@
 require('../common');
 const assert = require('assert');
 
-const { Transform, Readable, Writable } = require('stream');
+const { Transform, Readable, Writable, getDefaultHighWaterMark } = require('stream');
 
-const DEFAULT = 16 * 1024;
+const DEFAULT = getDefaultHighWaterMark();
```

Comment:
This is breaking in the tests. Can you have a look @ronag?

Reply:
Fixed